slow_tests/test_barostat.py

```diff
@@ -414,13 +406,14 @@ def test_molecular_ideal_gas():
         v_0 = sample_velocities(masses * unit.amu, temperature)
 
         # rescale the box to be approximately the desired box volume already
-        rescaler = CentroidRescaler(group_indices)
+        # rescaler = CentroidRescaler(group_indices)
```

Comment:
for intentionally commented-out lines, would it make sense to leave a line comment explaining when the code might be useful in the future and what will be the effect of uncommenting it?

Reply:
That was in error! Addressed in https://github.com/proteneer/timemachine/pull/598/commits/851bda71f746fd5a3de7702208fa5ad9c6b60005

(Had temporarily commented it out when checking if it was necessary for the test, forgot to delete after checking.)

Reply:
>for intentionally commented-out lines, would it make sense to leave a line comment explaining when the code might be useful in the future and what will be the effect of uncommenting it?

This sounds like a good practice!